www/i18n/en.json

```diff
@@ -180,40 +123,19 @@
         "to-label": "To Label",
         "show-all": "All Trips",
         "no-trips-found": "No trips found",
-        "for-current-filter": "for current filter. Show All to remove filters",
-        "scroll-to-load-more": "Scroll to load more",
-        "filter-display-status": "Displaying {{displayTripsLength}} / {{allTripsLength}} trips",
-        "filter-display-range": "{{currentEnd | date}} to {{currentStart | date:'medium' }}",
         "choose-mode": "Mode 📝 ",
         "choose-replaced-mode": "Replaces 📝",
         "choose-purpose": "Purpose 📝",
         "choose-survey": "Add Trip Details 📝 ",
         "select-mode-scroll": "Mode (👇 for more)",
         "select-replaced-mode-scroll": "Replaces (👇 for more)",
         "select-purpose-scroll": "Purpose (👇 for more)",
-        "list-pick-a-date": "Pick a date",
         "today": "Today",
         "no-more-travel": "No more travel to show",
         "show-more-travel": "Show More Travel",
         "show-older-travel": "Show Older Travel",
         "no-travel": "No travel to show",
-        "no-travel-hint": "To see more, change the filters above or go record some travel!",
-        "no-trips-today": "No trips recorded on this day"
-    },
-
-    "new_label_tour": {
```

Comment:
If we have to...
I admit I am sort of opinionated against walkthroughs in mobile apps. My thinking is that if an app needs a walkthrough to be comprehensible to users, it has bad UX and adding a walkthrough is treating the symptom not the cause.

If it's fairly minimal -- maybe not a formal 'walkthrough' but just a 'hint' that comes up when relevant (for example, when someone encounters their first orange label, we can trigger a popup that points out the checkmark button), I would look on that more favorably.
What would you think about a more minimal, "tips" or "hints" style of user guidance?

Reply:
I think the idea of user hints is great, that way the user sees new features as they're relevant. I would also be ok with a walkthrough that's only available on demand. I'm not sure if these keys can be reused since the screen has changed, I can try to keep as many as still apply and we can add more later?

Reply:
I agree with you on the goal of having the UI be intuitive.

That's why I have "I am not sure if we should pop it up on first load or just have a help button to allow users to launch on demand" - I know that a lot of apps popup the walkthrough on first load, and we used to as well, but then the walkthrough was confusing, so we turned it off, but left it as a "help" option.

I only brought it up because the e-bike team specifically mentioned not understanding that the check mark would be a  one-click confirmation of the inferred label assist and they are fairly young "digital natives".

I would be totally fine with a minimal "tips" or "hints" style of user guidance. We can also wait to see if the accessibility fixes make it better, and/or try to get feedback from cortez, which has users who are 55+ and so, at least stereotypically, find it harder to navigate tech.

Reply:
Yeah remove them, but file a new issue for walkthrough/hints.

Reply:
tracking this in [issue#935](https://github.com/e-mission/e-mission-docs/issues/953)